install/network/avahi.sh

```diff
@@ -22,6 +22,13 @@ AVAHI_SERVICE_PATH="/etc/avahi/services"
 )
 curl -fsSL $CONFIGURE_NETWORK_PATH/blueos.service > "${AVAHI_SERVICE_PATH}/blueos.service"
 
+AVAHI_DAEMON_CONFIG_PATH="cat /etc/avahi/avahi-daemon.conf"
+[ ! -e "${AVAHI_DAEMON_CONFIG_PATH}" ] && (
+    echo "Avahi daemon config file does not exist: ${AVAHI_DAEMON_CONFIG_PATH}"
+    exit 1
+)
+sed -i "s/#host-name=foo/host-name=blueos-avahi/g" ${AVAHI_DAEMON_CONFIG_PATH}
+
```

Comment:
This appears to be a bit weird, why are we failing if avahi is not available since is not required for the system to work (since now we have beacon).
Also, the `cat` will make all other lines to not work.
The correct would be:

```suggestion
AVAHI_DAEMON_CONFIG_PATH="/etc/avahi/avahi-daemon.conf"
AVAHI_HOST_NAME="blueos-avahi"
[ ! -d "${AVAHI_DAEMON_CONFIG_PATH}" ] && (
    echo "Avahi daemon config file found, changing avahi host name to: ${AVAHI_HOST_NAME}"
    sed -i "s/#host-name=/host-name=blueos-avahi/g" ${AVAHI_DAEMON_CONFIG_PATH}
) || echo "Avahi daemon config file not found in ${AVAHI_DAEMON_CONFIG_PATH}"
```

This also remove the old "foo" name to make the command more future proof.